install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -60,12 +60,29 @@ spec:
           {{- if .Values.global.trustDomain }}
             - --trust-domain={{ .Values.global.trustDomain }}
           {{- end }}
+          {{- if .Values.healthChecking }}
+            - --liveness-probe-path=/tmp/ca.liveness # path to the liveness health check status file
+            - --liveness-probe-interval=60s # interval for health check file update
+            - --probe-check-interval=15s    # interval for health status check
+          {{- end }}
+          {{- if .Values.healthChecking }}
+          livenessProbe:
+            exec:
+              command:
+              - /usr/local/bin/istio_ca
+              - probe
+              - --probe-path=/tmp/ca.liveness # path to the liveness health check status file
+              - --interval=125s # the maximum time gap allowed between the file mtime and the current sys clock
+            initialDelaySeconds: 60
```

Comment:
how come this one is 60s but the other is 5s?

Reply:
Nevermind I see now that it is what we had before

Reply:
the other one is wrong. I don't think sending 'versions' endpoint helps for liveness probe.

Reply:
If it is wrong then should we change it? I guess might as well keep it to minimize risk

Reply:
Yes, I agree to keep it to minimize the risk. We may remove the other one in the future.

Reply:
I removed anyway, to avoid confusion to users. There's no risk as I see here.